### PR TITLE
Frio: sticky aside on scroll and fixed on mobile

### PR DESCRIPTION
**Pull request by fabrixxm**
Adds a new library : http://leafo.net/sticky-kit/
On desktop, the aside bar sticks in viewport.
On mobile, the aside is fixed on top fullpage.

At line 231 in js/theme.js there was extra closing brackets that was causing syntax error.

Sorry for the extra diff on space->tabs / tabs->space changes.. I'll pay more attention on this next time.

### CODE REVIEW

File: view/theme/frio/js/theme.js [hunk 1]
```diff
@@ -609,132 +600,30 @@ String.prototype.rtrim = function() {
...
-	if (typeof elementId === "undefined") {
+// Scroll to a specific item and highlight it
+// Note: jquery.color.js is needed
+function scrollToItem(itemID) {
+	if( typeof itemID === "undefined")
 		return;
-	}
 
-	var $el = $(document.getElementById(elementId));
+	var elm = $('#'+itemID);
```
Comment (rabuzarus) [inline view/theme/frio/js/theme.js]: We couldn't use jQuery in this function. Because jQuery will not work with hubzilla guid (@ and dots in the guid)

**Comment by rabuzarus** [comment]
@fabrixxm don't know if something went wrong but on github I can't see `function fixscroll()` form the first commit anymore.
And it seems to delete some other js functions in `theme.js` (https://github.com/friendica/friendica/pull/3372/files)

File: view/theme/frio/js/theme.js (hunk 1 above)
Comment (Hypolite) [inline view/theme/frio/js/theme.js]: This looks like a reversion of my own changes about scrolling to hubzilla comments, see the documentation header block being removed.

**Comment by fabrixxm** [comment]
@rabuzarus the fixscroll() function isn't needed anymore because I've added the sitcky-kit library, which does the same work better.

For the other, I got the last theme.js from develop, but I could have done some big mistakes...
I'll check again...

File: view/theme/frio/js/theme.js (hunk 1 above)
Comment (fabrixxm) [inline view/theme/frio/js/theme.js]: I could have screwed up the "merge develop" pass..

File: view/theme/frio/js/theme.js [hunk 2]
```diff
@@ -212,23 +212,7 @@ $(document).ready(function(){
 		toggleDropdownText(this);
 	});
 
-	/* setup onoff widgets */
-	// Add the correct class to the switcher according to the input
-	// value (On/Off)
-	$(".toggle input").each(function(){
```
Comment (Hypolite) [inline view/theme/frio/js/theme.js]: I confirm this is still in the current develop, there was something very wrong with your develop merge.

**Pull request closed without merging**